skore/src/skore/sklearn/_estimator/metrics_accessor.py

```diff
@@ -161,6 +163,7 @@ def report_metrics(
                     scoring += ["_roc_auc", "_log_loss"]
             else:
                 scoring = ["_r2", "_rmse"]
+            scoring += ["fit_time", "predict_time"]
```

Comment:
I would have expected to pass the private function "_fit_time" and "_predict_time" but I need to check why is it working :)

Reply:
OK so we do some processing afterwards to get the right function. Maybe let's be inline why the previous function such that we have the name corresponding to the actual private function.